### PR TITLE
Update screenshots on docs intro page

### PR DESCRIPTION
**Pull request by ChristopherGS**
Updated screenshots to show new UI and removed Beta button

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -225,9 +225,13 @@ and it'll end up as structured data in our platform ready to be queried.
 For example, using data from the `User` model above, we could list users from the USA:
 
 ```sql
-SELECT attributes->'result'->>'name' as name, age(attributes->'result'->>'dob') as age
+SELECT
```
Comment (ChristopherGS) [inline docs/index.md]: when I ran this query, I got this error:
image

So I updated with a query that does work

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ __pycache__
 
 # stubgen
 out
+
+# intellij
+.idea
```
Comment (Kludex) [inline .gitignore]: I prefer for ppl to use their own global `.gitignore` on IDE specific files.

```suggestion
```

**Review by Kludex — APPROVED** [review]
Screenshots looks fine.

File: docs/index.md (hunk 1 above)
Comment (alexmojaki) [reply docs/index.md]: I personally significantly prefer the original example. I suggest:

```sql
SELECT attributes->'result'->>'name' as name, extract(year from (attributes->'result'->>'dob')::date) as "birth year"
FROM records
WHERE attributes->'result'->>'country_code' = 'USA'
```

to demonstrate some fancy-looking SQL. Calculating the age would be better, but apparently datafusion doesn't support extracting years from durations.

File: .gitignore (hunk 2 above)
Comment (alexmojaki) [reply .gitignore]: What's the harm? Does .gitignore need to be clean?

**Comment by alexmojaki** [comment]
It's not important, but I think future screenshots should probably have a profile pic in them.

File: docs/images/index/logfire-screenshot-search-query.png
Comment (alexmojaki) [inline docs/images/index/logfire-screenshot-search-query.png:1]: The mixture of with and without service names doesn't look very good on the left, nor does `unknown_service` on the right (although other people probably wouldn't notice that bit). It'd also be nice if the service names weren't truncated to `example-s...` and `example-a...`. How about just `api` and `worker`?

File: .gitignore (hunk 2 above)
Comment (Kludex) [reply .gitignore]: I wouldn't accept this on the projects I maintain cause it opens space for ppl to keep adding stuff. I prefer correctness. The gitignore for a project should only contain what is generated from the project itself but shouldn't be committed.

File: docs/index.md (hunk 1 above)
Comment (ChristopherGS) [reply docs/index.md]: updated

File: .gitignore (hunk 2 above)
Comment (ChristopherGS) [reply .gitignore]: removed

File: docs/images/index/logfire-screenshot-search-query.png
Comment (ChristopherGS) [reply docs/images/index/logfire-screenshot-search-query.png:1]: fixed

**Pull request merged**